src/lava/proc/scif/process.py

```diff
@@ -23,7 +23,8 @@ def __init__(
             shape: ty.Tuple[int, ...],
             step_size: ty.Optional[int] = 1,
             theta: ty.Optional[int] = 4,
-            neg_tau_ref: ty.Optional[int] = -5) -> None:
+            neg_tau_ref: ty.Optional[int] = -5,
+            noise_amplitude: ty.Optional[int] = 0) -> None:
```

Comment:
Wasn't this a part of the PR we already reviewed?

Reply:
Yes!